class.AttachmentPreviewPlugin.php

```diff
@@ -1057,8 +1057,9 @@ public static function isTicketsView() {
       $tickets_view = FALSE;
     }
     elseif (strpos($url, 'index.php') !== FALSE ||
-      strpos($url, 'tickets.php') !== FALSE) {
-      // Might be a ticket page..
+      strpos($url, 'tickets.php') !== FALSE) ||
```

Comment:
Syntax error with this brace ```)```, suggest removing it.

Reply:
yes, removing the `)` in line 1060 within `) ||` works - at least the system does not crash.